### PR TITLE
Update SAML 2.0 docs with Azure instructions

### PR DESCRIPTION
**Pull request by BrJan**
## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [ ] No: please ensure the base branch is `current`
- [ ] Unsure: we'll let you know!

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

### CODE REVIEW

**Comment by BrJan** [comment]
@barberscott take a look when you have chance. Thanks!

File: website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md [hunk 1]
```diff
@@ -224,6 +224,58 @@ the integration between Okta and dbt Cloud.
 After creating the Okta application, follow the instructions in the [dbt Cloud Setup](#dbt-cloud-setup)
 section to complete the integration.
 
+## Azure integration
+
+_Use this section if you are configuring Azure AD as your identity provider_.
+
+### Create Azure AD Enterprise Application
+
+First, log into your Azure account. Follow the steps below to set up single sign-on with dbt Cloud.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md]:
```suggestion
Follow these steps to set up single sign-on (SSO) with dbt Cloud:
```

File: website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md [hunk 2]
```diff
@@ -224,6 +224,58 @@ the integration between Okta and dbt Cloud.
...
+
+- In the Azure AD portal, choose **Enterprise applications** and create a new application.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md]:
```suggestion
1. Log into your Azure account. 
2. In the Azure AD portal, select **Enterprise applications** and click **+ New application**.
```

File: website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md [hunk 3]
```diff
@@ -224,6 +224,58 @@ the integration between Okta and dbt Cloud.
...
+- Select **Create your own application**.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md]:
```suggestion
3. Select **Create your own application**.
```

File: website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md [hunk 4]
```diff
@@ -224,6 +224,58 @@ the integration between Okta and dbt Cloud.
...
+
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md]:
```suggestion
```

File: website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md [hunk 5]
```diff
@@ -224,6 +224,58 @@ the integration between Okta and dbt Cloud.
...
+- Name the application **dbt Cloud** or some other descriptive name.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md]:
```suggestion
4. Name the application "dbt Cloud" or another descriptive name.
```

File: website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md [hunk 6]
```diff
@@ -224,6 +224,58 @@ the integration between Okta and dbt Cloud.
...
+- Select the **Non-gallery** option for application type.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md]:
```suggestion
5. Select **Integrate any other application you don't find in the gallery (Non-gallery)** as the application type.
```

File: website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md [hunk 7]
```diff
@@ -224,6 +224,58 @@ the integration between Okta and dbt Cloud.
...
+- When the application is done being created, find it in the **Enterprise applications > All applications** view.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md]:
```suggestion
6. Click **Create**.
7. You can find the new application by clicking **Enterprise applications** and selecting **All applications**.
```

File: website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md [hunk 8]
```diff
@@ -224,6 +224,58 @@ the integration between Okta and dbt Cloud.
...
+- Click the application to configure.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md]:
```suggestion
8. Click the application you just created and follow the instructions for configuring it in [Configuring SAML endpoints in AD](#configuring-saml-endpoints-in-ad). 
```

File: website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md [hunk 9]
```diff
@@ -224,6 +224,58 @@ the integration between Okta and dbt Cloud.
...
+
+### Configurating SAML Endpoints in AD
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md]:
```suggestion
### Configuring SAML endpoints in AD
```

File: website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md [hunk 10]
```diff
@@ -224,6 +224,58 @@ the integration between Okta and dbt Cloud.
...
+
+- Navigate to **Single sign-on > Set up single sign on**.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md]:
```suggestion
1. Select **Single sign-on** in the left navigation and click **Set up single sign on**.
```

File: website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md [hunk 11]
```diff
@@ -224,6 +224,58 @@ the integration between Okta and dbt Cloud.
...
+- Under **Select a single sign-on method**, choose **SAML**.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md]:
```suggestion
2. Click **SAML** in "Select a single sign-on method" section.
```

File: website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md [hunk 12]
```diff
@@ -224,6 +224,58 @@ the integration between Okta and dbt Cloud.
...
+- Edit the **Basic SAML Configuration**.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md]:
```suggestion
3. Click **Edit** in "Basic SAML Configuration." 
4. Use the following table to help you fill out the key fields required to connect to dbt:
```

File: website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md [hunk 13]
```diff
@@ -224,6 +224,58 @@ the integration between Okta and dbt Cloud.
...
+
+| Field | Value |
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md]: @BrJan can you indent the entire table so it's inline with the text of step 4?

File: website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md [hunk 14]
```diff
@@ -224,6 +224,58 @@ the integration between Okta and dbt Cloud.
...
+| ----- | ----- |
+| **Identifier (Entity ID)** | This is the base URL for dbt Cloud (https://cloud.getdbt.com/ or https://yourcompany.getdbt.com/ for a single tenant instance). |
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md]:
```suggestion
| **Identifier (Entity ID)** | Base URL for dbt Cloud. Use `https://cloud.getdbt.com/` or `https://YOUR_COMPANY.getdbt.com/` for a single tenant instance. |
```

File: website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md [hunk 15]
```diff
@@ -224,6 +224,58 @@ the integration between Okta and dbt Cloud.
...
+| **Reply URL (Assertion Consumer Service URL)** | https://cloud.getdbt.com/complete/saml or https://yourcompany.getdbt.com/complete/saml for a single tenant instance. |
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md]:
```suggestion
| **Reply URL (Assertion Consumer Service URL)** | Use `https://cloud.getdbt.com/complete/saml` or `https://YOUR_COMPANY.getdbt.com/complete/saml` for a single tenant instance. |
```

File: website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md [hunk 16]
```diff
@@ -224,6 +224,58 @@ the integration between Okta and dbt Cloud.
...
+| **Relay State** | This is the slug you will configure in dbt Cloud. It's usually your company name, but you can pick anything you'd like. |
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md]: Can we provide an example here?
```suggestion
| **Relay State** | The slug you will configure in dbt Cloud. It's usually your company name, but you can pick anything you'd like. |
```

File: website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md [hunk 17]
```diff
@@ -224,6 +224,58 @@ the integration between Okta and dbt Cloud.
...
+
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md:239]: 5. Click **Save** at the top of the form.

File: website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md [hunk 18]
```diff
@@ -224,6 +224,58 @@ the integration between Okta and dbt Cloud.
...
+#### Creating SAML Attributes in AD
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md:240]: Can you add a little about why they should do this and how to get here? I couldn't find this in AD.

File: website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md [hunk 19]
```diff
@@ -224,6 +224,58 @@ the integration between Okta and dbt Cloud.
...
+
+- Edit the **User Attributes & Claims**.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md]: Can you number the rest of the steps? It helps people remember where they're at and distinguished between optional unordered items, and actuals ordered steps if what you need to do to be successful.

**Review by runleonarun — COMMENTED** [review]
Hey @BrJan I'm super impressed with how organized and precise this is! Great work! Can Let me know if I can explain any of my suggestions.

File: website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md [hunk 20]
```diff
@@ -224,6 +224,60 @@ the integration between Okta and dbt Cloud.
...
+2. Click **SAML** in "Select a single sign-on method" section.
+3. Click **Edit** in "Basic SAML Configuration." 
+4. Use the following table to help you fill out the key fields required to connect to dbt:
+
+| Field | Value |
+| ----- | ----- |
+| **Identifier (Entity ID)** | Base URL for dbt Cloud. Use `https://cloud.getdbt.com/` or `https://YOUR_COMPANY.getdbt.com/` for a single tenant instance. |
+| **Reply URL (Assertion Consumer Service URL)** | Use `https://cloud.getdbt.com/complete/saml` or `https://YOUR_COMPANY.getdbt.com/complete/saml` for a single tenant instance. |
+| **Relay State** | The slug you will configure in dbt Cloud. It's usually your company name, but you can pick anything you'd like. |
+
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md:239]:
```suggestion
5. Click **Save** at the top of the form.
```

**Review by saimaddali — APPROVED** [review]

File: website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md (hunk 19 above)
Comment (runleonarun) [reply website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md]: Addressed in https://github.com/dbt-labs/docs.getdbt.com/pull/807/commits/bbbf230a75a03c21624c9e3118ffce3b97dee844

File: website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md [hunk 21]
```diff
@@ -219,39 +205,91 @@ the integration between Okta and dbt Cloud.
...
+   | **last_name** | user.surname |
+
+5. Click **Add a group claim** from User Attributes and Claims.
+6. If you'll assign users directly to the enterprise application, select **Security Groups**. If not, select **Groups assigned to the application**.
+7. Set **Source attribute** to **Group ID**.
+8. Under **Advanced options**, check **Customize the name of the group claim** and specify **Name** to **groups**.
+
+**Note:** Keep in mind that the Group ID in Azure AD maps to that group's GUID. It should be specified in lowercase for the mappings to work as expected. The Source Attribute field alternatively can be set to a different value of your preference.
+
 ### Finish setup
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md]:
```suggestion
```

File: website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md [hunk 22]
```diff
@@ -219,39 +205,91 @@ the integration between Okta and dbt Cloud.
...
 
-After creating the Okta application, follow the instructions in the [dbt Cloud Setup](#dbt-cloud-setup)
+After creating the Azure application, follow the instructions in the [dbt Cloud Setup](#dbt-cloud-setup)
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md]:
```suggestion
9. After creating the Azure application, follow the instructions in the [dbt Cloud Setup](#dbt-cloud-setup)
```

File: website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md [hunk 23]
```diff
@@ -219,39 +205,90 @@ the integration between Okta and dbt Cloud.
...
-| Log&nbsp;in&nbsp;with | SAML 2.0 |
-| Identity&nbsp;Provider&nbsp;SSO&nbsp;Url | Paste the **Identity Provider Single Sign-On URL** shown in the IdP setup instructions |
-| Identity&nbsp;Provider&nbsp;Issuer | Paste the **Identity Provider Issuer** shown in the IdP setup instructions |
-| X.509&nbsp;Certificate | Paste the **X.509 Certificate** shown in the IdP setup instructions |
-| Slug | Enter your desired login slug. |
+#### Creating SAML Attributes in AD
+
+From the Set up Single Sign-On with SAML page:
+
+1. Click *Edit* in the User Attributes & Claims section.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-saml-2.0.md]:
```suggestion
1. Click **Edit** in the User Attributes & Claims section.
```

**Comment by runleonarun** [comment]
Status: @mirnawong1 will give my edits a final review and possible add a sentence or two of context then we can merge this!

Brian, I know this PR has been outstanding for too long, but we're zeroing in on a process that will help us respond to PRs in a timely manner. This PR will be super helpful to people, so thank you for getting the core content in there!

**Pull request merged**